src/data/pnat/basic.lean

```diff
@@ -348,34 +370,92 @@ theorem lcm_dvd {m n k : ℕ+} (hm : m ∣ k) (hn : n ∣ k) : lcm m n ∣ k :=
 theorem gcd_mul_lcm (n m : ℕ+) : (gcd n m) * (lcm n m) = n * m :=
  subtype.eq (nat.gcd_mul_lcm (n : ℕ) (m : ℕ))
 
+section prime
+/-! ### Prime numbers -/
+
 def prime (p : ℕ+) : Prop := (p : ℕ).prime
 
-/-- Two pnats are coprime if their gcd is 1. -/
-def coprime (m n : ℕ+) : Prop := m.gcd n = 1
+lemma prime.one_lt {p : ℕ+} : p.prime → 1 < p := nat.prime.one_lt
+
+lemma prime_two : (2 : ℕ+).prime := nat.prime_two
+
+lemma dvd_prime {p m : ℕ+} (pp : p.prime) :
+(m ∣ p ↔ m = 1 ∨ m = p) := by { rw pnat.dvd_iff, rw nat.dvd_prime pp, simp }
+
+lemma prime.ne_one {p : ℕ+} : p.prime → p ≠ 1 :=
+by { intro pp, intro contra, apply nat.prime.ne_one pp, rw pnat.coe_eq_one_iff, apply contra }
+
+lemma eq_one_of_lt_two {n : ℕ+} : n < 2 → n = 1 :=
```

Comment:
This shouldn't be in the section on prime numbers. Please put it above that section.
Can you prove this with `:= dec_trivial`?

Reply:
I moved it, but dec_trivial didn't work. I also tried tactic.exact_dec_trivial, copying a line earlier in the same file, which also didn't work.